.wercker.yml

```diff
@@ -20,6 +20,7 @@ build:
         - script:
             name: Build and install the conda package.
             code: |-
+                conda install -y git
```

Comment:
Appears this was the magic ingredient. Maybe it is time to switch to `conda` packages for this stuff. Please see PR ( https://github.com/jakirkham/docker_centos_drmaa_conda/pull/23 ).